docs/SUMMARY.md

```diff
@@ -1,58 +1,63 @@
-# Table of contents
+## Onboarding
 
 * [Welcome 👋](README.md)
 * [Get started](get-started.md)
 * [Your first pipeline](first-pipeline.md)
-* [A real ML pipeline](real-example.md)
-* [The Sematic UI](sematic-ui.md)
-* [The Sematic CLI](cli.md)
-* [Types](types.md)
-* [Local and Cloud execution](execution-modes.md)
+* [The MNIST pipeline](real-example.md)
+* [Capabilities](capabilities.md)
 * [Sematic vs. ...](sematic-vs.md)
-* [Join us on Discord](https://discord.gg/4KZJ6kYVax)
 
 ## Diving deeper
 
+* [Concepts](concepts.md)
 * [Sematic Functions](functions.md)
-* [Type support](type-support.md)
+* [The CLI](cli.md)
+* [Local execution](local-execution.md)
+* [The Dashboard](sematic-ui.md)
 * [Future algebra](future-algebra.md)
-* [Deploy Sematic](deploy.md)
-* [Container images](container-images.md)
 * [Artifacts](artifacts.md)
-* [Advanced APIs](advanced-apis.md)
 * [Glossary](glossary.md)
 
-## Integrations
+## Cloud execution
 
-* [Snowflake](snowflake.md)
-* [Grafana](grafana.md)
-* [Apache Spark](integrations.md)
-* [Google Dataflow](dataflow.md)
-* [Google BigQuery](bigquery.md)
-* [AWS Redshift](redshift.md)
-* [DBT](dbt.md)
-* [Slack](slack.md)
+* [Deploy Sematic](deploy.md)
+* [Cloud resolver](cloud-resolver.md)
+* [Container images](container-images.md)
 
-## Contribute
+## Public API reference
 
-* [Code of Conduct](code-of-conduct.md)
-* [How to contribute](contributor-guide.md)
-    * [Example pipelines](contribute-example.md)
-* [Monorepo](monorepo.md)
-* [Code style](code-style.md)
+* [API reference](api.md)
 
-## Design
+## Type support
 
-* [Concepts](concepts.md)
-* [Architecture](architecture.md)
-* [Stack](stack.md)
-* [Tooling](tooling.md)
-
-## Project
+* [Type support](type-support.md)
+* [Additional types](types.md)
 
-* [Roadmap](https://trello.com/b/AznpHZ7e/sematic-public-roadmap)
-* [Changelog](changelog.md)
+## Integrations
 
-## Company
+<!---
+* Apache Spark
+* Ray
+-->
+* [Snowflake](snowflake.md)
+* [Grafana](grafana.md)
+<!---
+* Slack
+* PyTorch
+* Pandas
+* Plotly
+* Matplotlib
```

Comment:
Why are these 3 commented-out here, but listed in `docs/types.md`?

Reply:
Because I haven't gotten around to it yet. These need code examples and screenshots, i.e. more work.